samples/AppSecretsConfig/README.md

```diff
@@ -0,0 +1,251 @@
+    }
+  }
+}
+```
+
+### Building the sample
+
+You can build, run, and even deploy the sample using [Visual Studio][visualstudio], [Visual Studio Code][visualstudiocode], or the [.NET command line interface (CLI)][dotnet_cli]. Using the `dotnet` CLI, for example, within the directory containing the sample application source run:
+
+```bash
+dotnet build
+```
+
+### Running the sample
+
+Before you can run the sample locally you'll need to grant access to the Key Vault for your local user account or service principal. If you're logged into your tenant in Visual Studio and the Azure CLI using the same principal, you can find the UPN or SPN the same way:
+
+```bash
+az account show
+```
+
+If you're logged in as a user, you will see your email address and can pass that to the `--upn` parameter:
+
+```bash
+az keyvault set-policy -n {vault-host-name} --upn {user@domain.com} --secret-permissions get
+```
+
+If you're logged in as a service principal, the `user.type` will be `servicePrincipal` and you'll pass the the `user.name` to the `--spn` parameter:
+
+```bash
+az keyvault set-policy -n {vault-host-name} --spn {spn} --secret-permissions get
+```
+
+#### Visual Studio
+
+1. Right-click on the project
+2. Click **Managed User Secrets**
+3. Add a variable named `ConnectionStrings:AppConfig` with the `value` of the `appConfigurationConnectionString` output variable:
+
+   ```json
+   {
+     "ConnectionStrings:AppConfig": "Endpoint=https://{appconfig-host-name}.azconfig.io;Id={id};Secret={secret}"
+   }
+   ```
+
+4. Click **Debug -> Start debugging (F5)** to run.
+
+#### Visual Studio Code
+
+1. In the project folder, run the following to add a variable named `ConnectionStrings:AppConfig` with the `value` of the `appConfigurationConnectionString` output variable:
+
+   ```bash
+   dotnet user-secrets set "ConnectionStrings:AppConfig" "Endpoint=https://{appconfig-host-name}.azconfig.io;Id={id};Secret={secret}"
+   ```
+
+2. With a *.cs* file open the command palette and run `Debug: Start debugging` or press `F5` (default binding).
+3. If prompted, select ".NET Core" to create a launch configuration and start debugging.
+
+#### dotnet CLI
+
+1. In the project folder, run the following to add a variable named `ConnectionStrings:AppConfig` with the `value` of the `appConfigurationConnectionString` output variable:
+
+   ```bash
+   dotnet user-secrets set "ConnectionStrings:AppConfig" "Endpoint=https://{appconfig-host-name}.azconfig.io;Id={id};Secret={secret}"
+   ```
+
+2. Run the project:
+
+   ```bash
+   dotnet run
+   ```
+
+#### Deploying the sample
+
+See the [ASP.NET quickstart][aspnet_quickstart] for instructions to deploy for Visual Studio, Visual Studio Code, and the `dotnet` CLI. For Visual Studio and Visual Studio Code, make sure you select your existing resource if you deployed the Bicep template above; otherwise, for the `dotnet` CLI you can deploy to an existing resource by [configuring Local Git support][aspnet_deploy_localgit].
```

Comment:
```suggestion
See the [ASP.NET quickstart][aspnet_quickstart] for instructions to deploy for Visual Studio, Visual Studio Code, and the `dotnet` CLI. For Visual Studio and Visual Studio Code, make sure you select your existing resource if you deployed the Bicep template above; otherwise, for the `dotnet` CLI you can deploy to an existing resource by [configuring local Git support][aspnet_deploy_localgit].
```

Reply:
That was actually intentional, since that's what Azure calls it, though it might be "LocalGit". I'll double check. I suppose if it is separate words, the subtlety is probably lost and what you suggest may read better.